src/llvm/tools/clang/lib/Basic/Targets/MSP430.h

```diff
@@ -31,9 +31,10 @@ class LLVM_LIBRARY_VISIBILITY MSP430TargetInfo : public TargetInfo {
     TLSSupported = false;
     IntWidth = 16;
     IntAlign = 16;
-    LongWidth = 32;
-    LongLongWidth = 64;
-    LongAlign = LongLongAlign = 16;
+    LongWidth = FloatWidth = 32;
```

Comment:
What about data layout string?

Reply:
It's fine as I can see.
I've checked, it matches with backend one and represents data types and their alignment correctly.